lib/lich/gameobj.rb

```diff
@@ -0,0 +1,341 @@
+require 'rexml/document'
+
+# GameObj#load_data depends on this monkey patch
+class NilClass
+  def method_missing(*); end
+end
+
+# Tell GameObj where to find the GameObj data XML file
+class GameObj
```

Comment:
`GameObj.load_data` does take a filename argument so this isn't strictly necessary, but I thought this was a reasonable default for this repo.